ash/src/util.rs

```diff
@@ -59,6 +60,15 @@ impl<T> Align<T> {
         }
     }
 
+    pub unsafe fn index(&self, index: Range<usize>) -> Align<T> {
+        Align {
+            ptr: (self.ptr as *mut u8).offset((index.start as u64 * self.elem_size) as isize) as *mut c_void,
```

Comment:
Afaik `.offset` is implemented for `*mut c_void` and jumps by a byte-size of `1` - the cast to `*mut u8` and back to `*mut c_void` should not be necessary.